### DISCUSSION TITLE
Shizuku not working when device locked

### DISCUSSION THREAD

**Discussion by Sweep1422** [RikkaApps/Shizuku#596] [Q&A]
Shizuku doesn't work while my S23 is locked. I'm not sure if this is the expected behavior. It's automatically activated after boot through wifi ADB. I've tried following this [troubleshooting guide](https://shizuku.rikka.app/guide/setup/#start-via-wireless-debugging-start-by-connecting-to-a-computer-shizuku-randomly-stops), but the "Charge only" setting is being immediately reverted to debugging as soon as the device is locked. Maybe someone who is more knowledgeable about this could chime in. Thanks.

Edit: Managed to make it work by activating it through USB ADB. Though I would still appreciate it if someone knows of a way to achieve this wirelessly.

Edit2: For some reason it now also works with wifi ADB...

**Comment by virtuallysarvad** [comment]
I'm still facing this issue. Tried activating it through USB adb but as soon as I disconnect my device from charge cable it stops working
